packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -20,9 +20,15 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
 
 ### Creating users
 
+![Flowchart - create user](/img/oin/scim_flow-user-create.png "Simple flow diagram for create user process")
+
 The User creation operation brings the user's application profile from Okta over to the Service Provider. A user's application profile represents the key-value attributes defined on the **Profile** tab when a user is added.
 
-To enable user provisioning, an Okta administrator must configure the provisioning options in the Okta Admin Console. In the Okta Admin Console, select your SCIM application from your list of applications. Under the **Provisioning** tab, click **To App** and **Edit**. In the **Create User** option, click **Enable** and then **Save**.
+To enable user provisioning, an Okta administrator must configure the provisioning options in the Okta Admin Console. In the Okta Admin Console:
+
+* Select your SCIM application from your list of applications
```

Comment:
So, these are steps that need to be numbered and need periods at the end of each.

Reply:
Fixed